### PR TITLE
SchedulerConfigTest

### PR DESCRIPTION
**Pull request by apereverzin**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/BPS-970

### Change description ###
Added SchedulerConfigTest to integration tests.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/blobrouter/tasks/processors/ContainerCleaner.java [hunk 1]
```diff
@@ -33,9 +33,10 @@ public ContainerCleaner(
     public void process(String containerName) {
         logger.info("Started deleting dispatched blobs from container {}", containerName);
 
-        final BlobContainerClient containerClient = storageClient.getBlobContainerClient(containerName);
 
         try {
+            final BlobContainerClient containerClient = storageClient.getBlobContainerClient(containerName);
+
```
Comment (rkondratowicz) [inline src/main/java/uk/gov/hmcts/reform/blobrouter/tasks/processors/ContainerCleaner.java]: unrelated change?

Comment (apereverzin) [reply src/main/java/uk/gov/hmcts/reform/blobrouter/tasks/processors/ContainerCleaner.java]: Related - otherwise exception is thrown outside due to misconfiguration in integration tests

File: src/integrationTest/java/uk/gov/hmcts/reform/blobrouter/features/SchedulerConfigTest.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+@Profile("integration-test")
+public class SchedulerConfigTest {
+
+    @SpyBean
+    private LockProvider lockProvider;
+
+    @Test
+    public void should_integrate_with_shedlock() throws Exception {
+        ArgumentCaptor<LockConfiguration> configCaptor = ArgumentCaptor.forClass(LockConfiguration.class);
+        Thread.sleep(2000);
```
Comment (nitinprabhu) [inline src/integrationTest/java/uk/gov/hmcts/reform/blobrouter/features/SchedulerConfigTest.java]: why do we have sleep here @apereverzin ?

Comment (apereverzin) [reply src/integrationTest/java/uk/gov/hmcts/reform/blobrouter/features/SchedulerConfigTest.java]: Because the scheduled job runs asynchronously - in a background process

Comment (nitinprabhu) [reply src/integrationTest/java/uk/gov/hmcts/reform/blobrouter/features/SchedulerConfigTest.java]: cool worth putting a comment there
